tests/integration/test_api/test_statistics/test_statistics_format.py

```diff
@@ -0,0 +1,214 @@
+            brief: Wrapper of a helper function to handle Wazuh daemons.
+
+    assertions:
+        - Check if the statistics returned by the API have the expected format.
+
+    input_description:
+        - The `configuration_manager_statistics_format` file provides the module configuration for this test.
+        - The `cases_manager_statistics_format` file provides the test cases.
+    """
+    endpoint = test_metadata['endpoint']
+    statistics_schema_path = os.path.join(t1_statistics_template_path, f"{endpoint}_template.json")
+    params = f"?daemons_list={endpoint}"
+    url = get_base_url() + DAEMONS_STATS_ROUTE + params
+    authentication_headers, _ = login()
+
+    # Get daemon statistics
+    response = requests.get(url, headers=authentication_headers, verify=False)
+    stats_schema = read_json_file(statistics_schema_path)
+
+    # Check if the API statistics response data meets the expected schema. Raise an exception if not.
+    jsonschema.validate(instance=response.json(), schema=stats_schema)
+
+
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('test_metadata', test2_metadata, ids=test2_cases_ids)
+def test_agent_statistics_format(test_metadata, daemons_handler):
+    """
+    description: Check if the statistics returned by the API have the expected format.
+
+    test_phases:
+        - setup:
+            - Restart wazuh-manager service to apply configuration changes
+        - test:
+            - Simulate and connect an agent
+            - Request the statistics of a particular daemon and agent from the API
+            - Compare the obtained statistics with the json schema
+            - Stop and delete the simulated agent
+        - teardown:
+            - Stop wazuh-manager
+
+    wazuh_min_version: 4.4.0
+
+    parameters:
+        - test_metadata:
+            type: dict
+            brief: Get metadata from the module.
+        - daemons_handler:
+            type: fixture
+            brief: Wrapper of a helper function to handle Wazuh daemons.
+
+    assertions:
+        - Check if the statistics returned by the API have the expected format.
+
+    input_description:
+        - The `cases_agent_statistics_format` file provides the test cases.
+    """
+    endpoint = test_metadata['endpoint']
+    stats_schema_path = os.path.join(t2_statistics_template_path, f"{endpoint}_template.json")
+    agents = create_agents(1, 'localhost')
+    _, injector = connect(agents[0])
+
+    route = f"/agents/{agents[0].id}/daemons/stats"
+    params = f"?daemons_list={endpoint}"
+    url = get_base_url() + route + params
+    authentication_headers, _ = login()
+
+    # Get Agent's daemon stats
+    response = requests.get(url, headers=authentication_headers, verify=False)
+    stats_schema = read_json_file(stats_schema_path)
+
+    # Check if the API statistics response data meets the expected schema. Raise an exception if not.
+    jsonschema.validate(instance=response.json(), schema=stats_schema)
```

Comment:
Do this in the framework. Check for example: https://github.com/wazuh/qa-integration-framework/blob/17441-api-tests-migration/src/wazuh_testing/modules/analysisd/utils.py#L24

Reply:
Done